src/client/App.tsx

```diff
@@ -15,11 +15,25 @@ export default function App() {
   const [isNavBarOpen, setIsNavBarOpen] = React.useState<boolean>(true);
   const [page, setPage] = useState<Page>(Page.Clusters);
   const [metricSubPage, setMetricSubPage] = useState<SubPage>(SubPage.ClusterLoad);
+  const [docSubPage, setDocSubPage] = useState<SubPage>(SubPage.None);
+
+  const showApiDocPage: () => void = useCallback(() => setDocSubPage(SubPage.ApiDoc), [setDocSubPage]);
 
   return (
     <ThemeProvider theme={useLightTheme ? lightTheme : darkTheme}>
       <Stack id="app" direction='row'>
         <NavigationBar
+<<<<<<< HEAD
```

Comment:
This seems to be a merge conflict that has not been resolved but was committed.

Reply:
i think it might be because we merged that other pull request and it may have made changes to dev